src/docs/development/platform-integration/ios-app-clip.md

```diff
@@ -0,0 +1,375 @@
+
+### 2.3
+
+Enter your new target detail in the dialog.
+
+Select `Storyboard` for Interface.
+
+Select `UIKit App Delegate` for Life Cycle.
+
+{% include app-figure.md
+image="development/platform-integration/ios-app-clip/app-clip-details.png" %}
+
+### 2.4
+
+In the following dialog, activate the new scheme for the new target.
+
+{% include app-figure.md
+image="development/platform-integration/ios-app-clip/activate-scheme.png" %}
+
+## Step 3 - Remove unneeded files
+
+### Step 3.1
+
+In the Project Navigator, in the newly created App Clip group, delete
+everything except `Info.plist` and `<app clip target>.entitlements`.
+
+{{site.alert.tip}}
+For add-to-app users, it's up to the reader to decide how much of this template
+to keep to invoke `FlutterViewController` or `FlutterEngine` APIs from this code
+later.
+{{site.alert.end}}
+
+{% include app-figure.md
+image="development/platform-integration/ios-app-clip/clean-files.png" %}
+
+Move files to trash.
+
+### Step 3.2
+
+If you don't use the `SceneDelegate.swift`, remove reference to it in the
+`Info.plist`.
+
+Open the `Info.plist` file for the in the App Clip group. Delete the entire
+dictionary entry for `Application Scene Manifest`.
+
+{% include app-figure.md
+image="development/platform-integration/ios-app-clip/scene-manifest.png" %}
+
+## Step 4 - Share build configurations
+
+### 4.1
+
+Back in the project settings, select the project entry now rather than any
+targets.
+
+In the `Info` tab, under the `Configurations` expandable group, expand the
+`Debug`, `Profile`, and `Release` entries.
+
+For each, select the same value from the drop-down menu for the App Clip
+target as the entry selected for the normal app target.
+
+This lets your App Clip target have access to Flutter's required build settings.
+
+This step isn't necessary for add-to-app projects.
+
+{% include app-figure.md
+image="development/platform-integration/ios-app-clip/configuration.png" %}
+
+### 4.2
+
+In the App Clip group's `Info.plist` file, set:
+
+- `Build version string (short)` to `$(FLUTTER_BUILD_NAME)`
+- `Bundle version` to `$(FLUTTER_BUILD_NUMBER)`
+
+This step isn't necessary for add-to-app projects.
+
+## Step 5 - Share code and assets
+
+### Option 1 - Share everything
+
+Assuming the intent is to show the same Flutter UI in the standard app as in
+the App Clip, share the same code and assets.
+
+For each of the following: `Main.storyboard`, `Assets.xcassets`,
+`LaunchScreen.storyboard`, `GeneratedPluginRegistrant.m`, and
+`AppDelegate.swift`, select the file, then in the first tab of the inspector,
+also include the App Clip target in the `Target Membership` checkbox group.
+
+{% include app-figure.md
+image="development/platform-integration/ios-app-clip/add-target-membership.png"
+%}
+
+### Option 2 - Customize Flutter launch for App Clip
+
+In this case, do not delete everything in [Step 3](#step-3). Use the scaffolding
+and the [iOS add-to-app APIs] to do a custom launch of Flutter, such as to
```

Comment:
to do => to create...

Flutter, such => Flutter. For example see the [custom Flutter route][] example.